src/views/DissolutionFirm/DissolutionFirm.vue

```diff
@@ -287,6 +291,33 @@ export default class DissolutionFirm extends Mixins(DateMixin) {
   private get isCertifyInvalid () {
     return this.getValidateSteps && (!this.getCertifyState.certifiedBy || !this.getCertifyState.valid)
   }
+  // addition label if its SP/GPs
+  private get additionalLabel () {
+    let label
+    if (this.isTypeFirm) { // if Sp/GP
+      label = this.isTypeSoleProp ? 'Proprietor' : 'Partners'
+    }
+    return label
+  }
+
+  /**
+  Get additional value
+  if SP return proprietor email id
+  if GP return partner email ids (comma separated)
+  **/
+  private get additionalValue () {
+    let emailList
+    if (this.isTypeFirm) { // if Sp/GP
+      const roleType = this.isTypeSoleProp ? RoleTypes.PROPRIETOR : RoleTypes.PARTNER
+      const partnerDetails =
+      this.getParties?.filter(people => people.roles.some(role => role.roleType === roleType))
```

Comment:
Indent.

Reply:
I was thinking about indenting this, since it's a continuation of the previous line.

I see you put it all on 1 line. That's fine too :)